src/test/java/org/synyx/urlaubsverwaltung/absence/AbsenceTest.java

```diff
@@ -8,50 +8,42 @@
 
 import java.time.Clock;
 import java.time.LocalDate;
-import java.time.LocalDateTime;
-import java.time.ZoneId;
+import java.time.ZoneOffset;
 import java.time.ZonedDateTime;
-import java.util.List;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.synyx.urlaubsverwaltung.absence.AbsenceType.HOLIDAY_REPLACEMENT;
-import static org.synyx.urlaubsverwaltung.person.Role.USER;
 
-
-/**
- * Unit test for {@link Absence}.
- */
 class AbsenceTest {
 
     private Person person;
-    private AbsenceTimeConfiguration timeConfiguration;
+    private AbsenceTimeConfiguration absenceTimeConfiguration;
     private final Clock clock = Clock.systemUTC();
-    private final TimeSettings timeSettings = new TimeSettings();
 
     @BeforeEach
     void setUp() {
 
         person = new Person("muster", "Muster", "Marlene", "muster@example.org");
+        person.setId(10L);
 
+        final TimeSettings timeSettings = new TimeSettings();
         timeSettings.setTimeZoneId("Etc/UTC");
         timeSettings.setWorkDayBeginHour(8);
         timeSettings.setWorkDayEndHour(16);
-
-        timeConfiguration = new AbsenceTimeConfiguration(timeSettings);
+        absenceTimeConfiguration = new AbsenceTimeConfiguration(timeSettings);
     }
```

Comment:
Should we additionally test with a second time zone other than "Etc/UTC"? Maybe just one case to ensure it's taken into account?